core/arch/arm/plat-stm32mp2/conf.mk

```diff
@@ -0,0 +1,74 @@
+flavor_dts_file-257F_EV = stm32mp257f-ev1.dts
+
+flavorlist-MP25 = $(flavor_dts_file-257F_EV)
+
+ifneq ($(PLATFORM_FLAVOR),)
+ifeq ($(flavor_dts_file-$(PLATFORM_FLAVOR)),)
+$(error Invalid platform flavor $(PLATFORM_FLAVOR))
+endif
+CFG_EMBED_DTB_SOURCE_FILE ?= $(flavor_dts_file-$(PLATFORM_FLAVOR))
+endif
+CFG_EMBED_DTB_SOURCE_FILE ?= stm32mp257f-ev1.dts
+
+ifneq ($(filter $(CFG_EMBED_DTB_SOURCE_FILE),$(flavorlist-MP25)),)
+$(call force,CFG_STM32MP25,y)
+endif
+
+ifeq ($(filter $(CFG_STM32MP25),y),)
+$(error STM32 Platform must be defined)
+endif
+
+$(call force,CFG_ARM64_core,y)
+$(call force,CFG_WITH_LPAE,y)
+# arm-v8 platforms
+include core/arch/arm/cpu/cortex-armv8-0.mk
+$(call force,CFG_WITH_ARM_TRUSTED_FW,y)
+# Uncomment the line below to only support 64bit TA
+supported-ta-targets ?= ta_arm64
+
+$(call force,CFG_ARM_GIC_PM,y)
+$(call force,CFG_GIC,y)
+$(call force,CFG_INIT_CNTVOFF,y)
+$(call force,CFG_PM,y)
+$(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
+$(call force,CFG_DT,y)
+$(call force,CFG_GENERIC_BOOT,y)
+$(call force,CFG_DRIVERS_CLK,y)
+$(call force,CFG_DRIVERS_CLK_DT,y)
+$(call force,CFG_DRIVERS_CLK_EARLY_PROBE,y)
+$(call force,CFG_TEE_CORE_EMBED_INTERNAL_TESTS,n)
+
```

Comment:
what about sorting this `$(call force,...)` config list (lines 29 to 39) in alphabetical order?
and also move lines 21, 22 and 25 into this forced config list.